### DISCUSSION TITLE
Logon failure status when connecting to SMB share

### DISCUSSION THREAD

**Discussion by moosefuel** [iterate-ch/cyberduck#15315] [Q&A] [answered]
So I have to have a client connect to my SMB share over the internet from a Windows machine (please don't ask me why!). Since Windows doesn't understand how to connect to a non-standard SMB port, I've installed Cyberduck on their machine (because it allows you to specify the port).

The SMB share the client wants to connect to is hosted on macOS, and Mac computers can connect to it easily using this address: 

```
smb://username@ip-address:custom-port/sharename
```

However, when I use Cyberduck and enter the same info, it appears to contact the server fine, but rejects the username and/or password. Here's the error.

<img width="459" alt="Screenshot 2023-11-02 at 15 46 45" src="https://github.com/iterate-ch/cyberduck/assets/111374276/23999e7f-54a5-4bc9-ab02-57385cda5b74">

I noticed that in Cyberduck it says to enter WORKGROUP\username. I'm not quite sure, not being a Windows guy myself, what this means I have to do. Obviously I'm doing something wrong, please enlighten me on how to connect. 

<img width="467" alt="Screenshot 2023-11-02 at 15 50 03" src="https://github.com/iterate-ch/cyberduck/assets/111374276/d847ad75-db79-4edb-bd0e-bb3e01fb718d">

P.S., I took the above screenshot for demonstration purposes only.

**Comment by dkocher** [comment]
Try to reach out to the server administrator to find out what _Realm_ [^1] should be used instead of the default `WORKGROUP` which may cause the authentication failure.

[^1]: https://docs.cyberduck.io/protocols/smb/#authentication

**Reply by moosefuel** [reply]
Thank you. Since I am the server admin in this case, how do I find out what a realm is? Apologies, Windows folk, this is jargon all new to me coming from the Mac side.

**Comment by moosefuel** [comment, marked as answer]
OK problem solved. 

In Macos there is one more step you need to take to make a share Windows-friendly. In the Share->File Sharing system pref. pane you right click and get extra settings, where you can click to enable certain users to log on using "less secure" Windows credentials.

**Reply by dkocher** [reply]
In general refer to [Share Mac files with Windows users
](https://support.apple.com/en-am/guide/mac-help/mchlp1657/mac).
